platform/ios/test/MGLMapViewConversionTests.m

```diff
@@ -0,0 +1,52 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+#import "../../ios/src/MGLMapView.h"
```

Comment:
Shouldn’t the Mapbox.h import statement above have already taken care of importing MGLMapView.h?